src/services/jwt.services.js

```diff
@@ -0,0 +1,46 @@
+const jwt = require('jsonwebtoken');
+const { userDto } = require('../dto/user.dto');
+
+const sign = (payload, key, expiresIn) => jwt.sign(payload, key, { expiresIn });
+
+const verify = (payload, key) => {
+  try {
+    return jwt.verify(payload, key);
+  } catch {
+    return null;
+  }
+};
+
+const generateAccessToken = (payload) =>
+  sign(payload, process.env.JW_ACCESS_KEY, '10m');
```

Comment:
Ensure that `process.env.JW_ACCESS_KEY` is correctly set in your environment variables to avoid issues with token generation.